boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -580,4 +648,150 @@ TEST_F(
+    // the drive state.
+    App_CanRx_DIM_SWITCHES_SetSignal_START_SWITCH(
+        can_rx_interface, CANMSGS_DIM_SWITCHES_START_SWITCH_ON_CHOICE);
+    LetTimePass(state_machine, 10);
+    EXPECT_EQ(
+        App_GetDriveState(),
+        App_SharedStateMachine_GetCurrentState(state_machine));
+
+    // Introduce inverter fault, expect transition to fault state
+    App_CanRx_INVL_INTERNAL_STATES_SetSignal_D1_VSM_STATE_INVL(
+        can_rx_interface,
+        CANMSGS_INVL_INTERNAL_STATES_D1_VSM_STATE_INVL_BLINK_FAULT_CODE_STATE_CHOICE);
+    LetTimePass(state_machine, 10);
+    EXPECT_EQ(
+        App_GetFaultState(),
+        App_SharedStateMachine_GetCurrentState(state_machine));
+}
+
+TEST_F(DcmStateMachineTest, init_to_fault_state_on_right_inverter_fault)
+{
+    LetTimePass(state_machine, 10);
+    EXPECT_EQ(
+        App_GetInitState(),
+        App_SharedStateMachine_GetCurrentState(state_machine));
+
+    // Introduce inverter fault, expect transition to fault state
+    App_CanRx_INVR_INTERNAL_STATES_SetSignal_D1_VSM_STATE_INVR(
+        can_rx_interface,
+        CANMSGS_INVR_INTERNAL_STATES_D1_VSM_STATE_INVR_BLINK_FAULT_CODE_STATE_CHOICE);
+    LetTimePass(state_machine, 10);
+    EXPECT_EQ(
+        App_GetFaultState(),
+        App_SharedStateMachine_GetCurrentState(state_machine));
+}
+
+TEST_F(DcmStateMachineTest, drive_to_fault_state_on_right_inverter_fault)
+{
+    SetInitialState(App_GetDriveState());
+    // Turn the DIM start switch on to prevent state transitions in
+    // the drive state.
+    App_CanRx_DIM_SWITCHES_SetSignal_START_SWITCH(
+        can_rx_interface, CANMSGS_DIM_SWITCHES_START_SWITCH_ON_CHOICE);
+    LetTimePass(state_machine, 10);
+    EXPECT_EQ(
+        App_GetDriveState(),
+        App_SharedStateMachine_GetCurrentState(state_machine));
+
+    // Introduce inverter fault, expect transition to fault state
+    App_CanRx_INVR_INTERNAL_STATES_SetSignal_D1_VSM_STATE_INVR(
+        can_rx_interface,
+        CANMSGS_INVR_INTERNAL_STATES_D1_VSM_STATE_INVR_BLINK_FAULT_CODE_STATE_CHOICE);
+    LetTimePass(state_machine, 10);
+    EXPECT_EQ(
+        App_GetFaultState(),
+        App_SharedStateMachine_GetCurrentState(state_machine));
+}
+
+// DCM-20
+TEST_F(DcmStateMachineTest, init_to_fault_state_on_motor_shutdown_error)
+{
+    LetTimePass(state_machine, 10);
+    EXPECT_EQ(
+        App_GetInitState(),
+        App_SharedStateMachine_GetCurrentState(state_machine));
+
+    // Introduce motor shutdown error, expect transition to fault state
+    App_SharedErrorTable_SetError(
+        error_table, BMS_MOTOR_SHUTDOWN_DUMMY_MOTOR_SHUTDOWN, true);
+    LetTimePass(state_machine, 10);
+    EXPECT_EQ(
+        App_GetFaultState(),
+        App_SharedStateMachine_GetCurrentState(state_machine));
+}
+
+// DCM-20
+TEST_F(DcmStateMachineTest, drive_to_fault_state_on_motor_shutdown_error)
+{
+    SetInitialState(App_GetDriveState());
+    // Turn the DIM start switch on to prevent state transitions in
```

Comment:
```suggestion
    SetInitialState(App_GetDriveState());
    
    // Turn the DIM start switch on to prevent state transitions in
```

super nit;

Reply:
Did you just add a new line? lollll

Reply:
you'll have to commit the suggestion if you want the new line to be added

Reply:
fixed